Automattic/jetpack
# Social | Initial state: Migrate feature flags to script data

## Proposed changes:

* Migrate the remaining feature flags to the new script data

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add `define( 'JETPACK_SOCIAL_HAS_EDITOR_PREVIEW', true );` to `wp-config.php`
* Goto the post editor and open Jetpack/Social sidebar
* Confirm that you see the "Preview social posts" button
* Remove the constant defined above
* Confirm that you don't see the button
* Add/Remove the corresponding block sticker - `social-editor-preview`.
* Confirm that it works as expected.
* Confirm that the connection management links and buttons work as expected both in Jetpack and Simple sites
* Confirm that the social recommendation links in Jetpack dashboard point to the correct location
* Confirm that the share status UI is displayed as per the feature flag.
